src/index.html

```diff
@@ -9,7 +9,69 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>HTML Form</h1>
+  <fieldset class="info">
+    <legend>Personal information</legend>
+    <label class="surname" for="surname">Surname:</label>
+    <input type="text" id="surname" name="surname" autocomplete="off"
```

Comment:
```suggestion
    <input 
      type="text" 
      id="surname" 
      name="surname" 
      autocomplete="off"
    />
```
write tags with multiple attributes like this